### ISSUE TITLE
Swiftlint fails when no swift files found despite no issues found

### ISSUE THREAD

**Issue by EfficientSetting** [wearerequired/lint-action#89]
I apologize for the weird title and this weird issue, but here's my situation:

I've added this action to all our repos and to our template so that the action passing becomes a required step.
Both swiftlint and swiftformat are being checked, but for some repos, the transition from Objective-C to Swift hasn't started and so they don't have swift files yet.

Now the behavior here is different for both linters, which elevates this to an issue or at least enhancement.
Swiftlint fails with the following error 

```
No lintable files found at paths: ''
SwiftLint found no issues (failure)
```

while Swiftformat reports the following:

```
warning: No eligible files found at /Users/buildserver/actions-runner/_work/..
SwiftFormat completed in 0.01s.
0 files formatted.
SwiftFormat found no issues (success)
```

It seems like each linter reports success and failure differently. It would be great if this could be straightened out across the both linters to interpret no lintable files as success (since no issues)

**Comment by ocean90** [comment]
Hi, thanks for reporting this!

I think this is something that should be handled by the linter itself. SwiftLint should either not return an error in this case or provide an argument to change this. I know that stylelint has a similar behaviour but also has a [`--allow-empty-input` flag](https://stylelint.io/user-guide/usage/cli#--allow-empty-input---aei) to ignore no files as an error.

I'm not familiar with SwiftLint but [looking at the source](https://github.com/realm/SwiftLint/blob/4083f1688556fb55c3a4972f69bb95ea0405ac96/Source/swiftlint/Extensions/Configuration%2BCommandLine.swift#L63-L66) and there appears to be something called [`allowZeroLintableFiles`](https://github.com/realm/SwiftLint/search?q=allowZeroLintableFiles&unscoped_q=allowZeroLintableFiles). Is this a config you can set?

**Comment by EfficientSetting** [comment]
@ocean90 
I agree and usually that's the way I would want things to go, unfortunately often times it's hard to change such behavior on an established tool.

Anyway's I looked at `allowZeroLintableFiles` and that's actually what I was looking for. See https://github.com/realm/SwiftLint/pull/2732/files

I had no idea that existed and there's no documentation outside the sources for this... thanks! I'll close this
